skunkworks_crow/src/main/java/org/odk/share/fragments/BlankFormsFragment.java

```diff
@@ -188,8 +190,12 @@ public void toggle() {
     private void toggleButtonLabel() {
         if (selectedForms.size() == formAdapter.getItemCount()) {
             toggleButton.setText(getString(R.string.clear_all));
+            filledcount = selectedForms.size();
+            sendButton.setText("Send(" + (selectedForms.size() + FilledFormsFragment.instancescount) + ")");
```

Comment:
Why counting different tab instances as well to the current tab instance when we tap on send it won't send both filled forms and blank forms together. At a time either filled form instances can be sent or blank form instance can be sent.

Reply:
Ok then! Then I will make counter for two forms separately. I confused and make the count for adding 2 forms